### ISSUE TITLE
Fix error `Money::Currency::UnknownCurrency: Unknown currency ''` on setup

### ISSUE THREAD

**Issue by shen-sat** [openfoodfoundation/openfoodnetwork#7626]
### What we should change and why (this is tech debt)
This error is interesting: it pops up once when running `script/setup`, but after either:
- a fresh re-install of the repo, or
- replacing `Money.default_currency = Money::Currency.new(Spree::Config[:currency])` with `Money.default_currency = Money::Currency.new('AUD')` and then running the `script/setup` and then reverting the change the error disappears 🤔  

### Context
https://openfoodnetwork.slack.com/archives/C2GQ45KNU/p1620397593134200
Linked issue: #7613

### Impact and timeline
Timeline: 1 week. Impact: better onboarding experience for new developers ✨

**Comment by viktorsmari** [comment]
Could it be related to cache?

What if you try doing `rails cache:clear` in between?

**Comment by shen-sat** [comment]
Ah the mystery deepens! Thanks for the input @viktorsmari ! I'll try clearing my rails cache and see if the error re-appears 👍 

I didn't know about the second point (default currency switching from "USD" to "AUD"). It points perhaps to a bigger problem: that we have two configs rather than one. For this ticket I guess we should fix only the default currency error and either always use "USD"  or "AUD". But I'm aslo wondering if we should raise a separate ticket to merge all configs into one..? 🤔

**Comment by psychoslave** [comment]
So with @apricot12 we made a videoconference to talk about reproducibility of this issue. Here is a way to currently reproduce it systematically:
```bash
cd /tmp/
git clone $SOME_OFN_REPOSITORY_CLONE
cd /tmp/openfoodnetwork
bundle exec rails server
./script/setup
```

So, much thanks to @apricot12 who excavated the reproduction process. :pray:

**Comment by andrewpbrett** [comment]
Closed via #7664
